E1_TFHub_Sample_Deploy/export.py

```diff
@@ -43,22 +46,27 @@ def train_step(model, loss_fn, optimizer_fn, metric, image, label):
   metric(loss_)
 
 
-@tf.function
-def test(image, label):
-  preds = model(image)
-  label_onehot = tf.one_hot(label, 10)
-
-
-def main():
+def train_and_export(
```

Comment:
This is a good time to add [docstrings](https://github.com/tensorflow/hub/blob/master/examples/text_embeddings/export.py#L100) to exported functions.

Reply:
Done :)